### ISSUE TITLE
Bugfix: Country Report `tidyverse` dependency

### ISSUE THREAD

**Issue by mattroumaya** [Gilead-BioStats/gsm#1330]
## Expected Behavior
Local installation of `{gsm}` should render country report.

## Current Behavior
Local installation of `{gsm}` throws a tidyverse dependency issue when rendering the report since `tidyverse` isn't a `{gsm}` dependency. 

## Resolution
Modify the line [here](https://github.com/Gilead-BioStats/gsm/blob/dev/inst/report/KRIReportByCountry.Rmd#L20) to use specific packages within `tidyverse` explicitly.

**Comment by mattroumaya** [comment]
Same issue with the QTL report locally:

- This specific issue can probably be resolved by including `library(purrr)` and using `purrr::discard()`. 

```
processing file: KRIReportByQTL.Rmd
  |................................                                                                                                                                                                                  |  15% [unnamed-chunk-1]
Quitting from lines 19-38 [unnamed-chunk-1] (KRIReportByQTL.Rmd)
Error in `discard()`:
! could not find function "discard"
Backtrace:
 1. ... %>% unique()
 2. base::unique(.)
```

**Comment by mattroumaya** [comment]
Closed via #1331
